spec/models/instance_spec.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Instance do
+  describe 'Scopes' do
+    before { described_class.refresh }
+
+    describe '#searchable' do
+      before do
+        Fabricate :account, domain: 'host.example'
+        Fabricate :account, domain: 'other.example'
+        Fabricate :domain_block, domain: 'other.example'
+      end
+
+      it 'returns records not domain blocked' do
+        results = described_class.searchable
+
+        expect(results)
+          .to include(expected_instance)
+          .and not_include(not_expected_instance)
+      end
+
+      def expected_instance
+        described_class.where(domain: 'host.example').first
+      end
+
+      def not_expected_instance
+        described_class.where(domain: 'other.example').first
+      end
```

Comment:
Why not `find_by` instead of `where(…).first`? Alternatively, why not having the `expect` on `results.pluck(:domain)` and testing the values directly, cutting down on helper methods?